packages/repository/src/relations/has-many/has-many-through.repository.ts

```diff
@@ -141,36 +145,105 @@ export class DefaultHasManyThroughRepository<
       throughData?: DataObject<ThroughEntity>;
       throughOptions?: Options;
     },
-  ): Promise<void> {
-    throw new Error('Method not implemented.');
+  ): Promise<TargetEntity> {
+    const targetRepository = await this.getTargetRepository(); //{id: inq:[]}
+    const throughRepository = await this.getThroughRepository(); // {category: 1}
+    const targetInstance = await targetRepository.create(
```

Comment:
I feel there should be a try catch here. Say if the targetInstance creation fails, e.g. it already exists, then the rest of code should stop and return proper error.

Reply:
I think repositories themselves ( through repo and target here) would throw error if anything goes wrong. Do we need extra try catch for them?